pontis-ui/src/hooks/oracle.ts

```diff
@@ -9,12 +9,16 @@ import { bigNumberishArrayToDecimalStringArray } from "starknet/utils/number";
 export const AssetKeys = [
   "eth/usd",
   "btc/usd",
-  "luna/usd",
   "sol/usd",
+  "luna/usd",
   "avax/usd",
   "doge/usd",
   "shib/usd",
   "temp/usd",
+  "dai/usd",
+  "usdt/usd",
+  "usdc/usd",
+  "tusd/usd",
   "eth/mxn",
   "btc/usd-20220624",
   "btc/usd-20220930",
```

Comment:
also merge in master, that has updated futures

Reply:
Found the corresponding keys on the website. Didn't know where to look in `master`

Reply:
https://github.com/42labs/Pontis/blob/master/pontis-package/pontis/publisher/assets.py